packages/core/src/plugins/prettier.test.js

```diff
@@ -22,4 +22,22 @@ describe('prettier', () => {
     )
     expect(result).toBe('const foo = <div />;\n')
   })
+
+  it('should resolve the prettier config with the editorconfig option', async () => {
+    // only mock prettier `resolveConfig` method
+    jest.mock('prettier', () =>
```

Comment:
This will get hoisted to the top of the file and might have unintended side-effects for other tests importing Prettier. [`jest.doMock()`](https://facebook.github.io/jest/docs/en/jest-object.html#jestdomockmodulename-factory-options) is more appropriate for this case.